docs/guides/connect-to-state-with-url-hash.md

```diff
@@ -9,13 +9,13 @@ If you want to connect state of a store to URL hash, you can create your own has
 
 ```ts
 import { create } from 'zustand'
-import { persist, StateStorage } from 'zustand/middleware'
+import { persist, StateStorage, createJSONStorage } from 'zustand/middleware'
 
 const hashStorage: StateStorage = {
   getItem: (key): string => {
     const searchParams = new URLSearchParams(location.hash.slice(1))
     const storedValue = searchParams.get(key)
-    return JSON.parse(storedValue)
+    return storedValue ? JSON.parse(storedValue) : ''
```

Comment:
I think a more concise and single-responsibility-per-line-esque solution will be:

```ts
const storedValue = searchParams.get(key) ?? ''
return JSON.parse(storedValue)
```

Reply:
Sure, thanks for the suggestion!